packages/material-ui/src/styles/createPalette.js

```diff
@@ -58,8 +58,8 @@ export const dark = {
   },
   divider: 'rgba(255, 255, 255, 0.12)',
   background: {
-    paper: grey[800],
-    default: '#303030',
+    paper: '#121212',
+    default: '#121212',
```

Comment:
Sorry for the question but is it expected to have such a dark background color  as the default on the default dark theme? It doesn't look right.

Reply:
Yes, this color comes from the Material Design specification: https://material.io/design/color/dark-theme.html#properties

Reply:
I do agree with this general comment, it does look a bit strange. I'm saving this feedback. We have a designer joining the team full-time in the new few weeks.

- Material Design: `#121212` brightness 7%
- YouTube Music: `#040404` brightness 1%
- YouTube: `#131313` brightness 9%
- Facebook: `#131314` brightness 10%
- Chakra: `#141821` brightness 17%
- HeadlessUI: `#0E131D` brightness 15%
- linear.app product: `#17181B` brightness 14%, linear.app homepage `#070707` brightness 2%
- Twitter: `#000000` brightness 0%, Dimmed `#111820` 16%
- GitHub: `#0C0E12` brightness 9%, Dimmed `#1A1D22` 18%
- StackOverflow `#222222`brightness 18%

It also seems that the documentation should use the default values of the palette.

Reply:
wow this is dark..

Reply:
I was also [initially shocked](https://github.com/mui-org/material-ui/issues/25629) at the difference, but I've actually come to quite like the new Material defaults. I assume it also makes extra sense for newer fancy screens which can optimize energy usage for dark pixels.

I imagine it would be useful though (if it doesn't already exist) to be able to specify a base brightness and have the rest of the elevation levels automatically scale, similar to how color palettes can automatically infer dark/light based on tonalOffset.

Reply:
> I imagine it would be useful though (if it doesn't already exist) to be able to specify a base brightness and have the rest of the elevation levels automatically scale, similar to how color palettes can automatically infer dark/light based on tonalOffset.

Done in #25522

Reply:
Ahh I guess I should have looked at the actual code :P, I see the backgroundImage alphaOverlay now. Awesome stuff!